RecoMuon/MuonIdentification/python/MuonShowerDigiFiller_cfi.py

```diff
@@ -0,0 +1,11 @@
+import FWCore.ParameterSet.Config as cms
+
+MuonShowerDigiFillerBlock = cms.PSet(
+  ShowerDigiFillerParameters = cms.PSet(
+      digiMaxDistanceX = cms.double(25.0),
+      dtDigiCollectionLabel  = cms.InputTag("muonDTDigis"),
+      cscDigiCollectionLabel = cms.InputTag("muonCSCDigis","MuonCSCStripDigi")
```

Comment:
@battibass @slava77 it looks like this line is creating issues in wf 1102.0 step4 RECOFROMRECO:

```
   [7] Prefetching for module TrackProducer/'mergedDuplicateTracks'
   [8] Prefetching for module DuplicateTrackMerger/'duplicateTrackCandidates'
   [9] Prefetching for module TrackCollectionMerger/'preDuplicateMergingGeneralTracks'
   [10] Prefetching for module TrackProducer/'muonSeededTracksInOut'
   [11] Prefetching for module CkfTrackCandidateMaker/'muonSeededTrackCandidatesInOut'
   [12] Prefetching for module MuonReSeeder/'muonSeededSeedsInOut'
   [13] Calling method for module MuonIdProducer/'earlyMuons'
Exception Message:
Principal::getByToken: Found zero products matching all criteria
Looking for type: MuonDigiCollection&lt;CSCDetId,CSCStripDigi&gt;
Looking for module label: muonCSCDigis
Looking for productInstanceName: MuonCSCStripDigi

   Additional Info:
      [a] If you wish to continue processing events after a ProductNotFound exception,
add "SkipEvent = cms.untracked.vstring('ProductNotFound')" to the "options" PSet in the configuration.

----- End Fatal Exception -------------------------------------------------
```

This clearly needs to be fixed, my question is: if this is not affecting any other standard workflow as it seems, and it is not preventing the basic validation of pre4, can we consider to postpone its fix after pre4 is built? Or do you see further possible issues?

Switching this module off in the configuration allows also the step4 of 1102 to run without problems.

Reply:
@fabiocos , @slava77 can surely reply more accurately than I can. 
In principle this PR implies that digis are available (hence implies that unpacking was done starting from RAW). For any workflow for which this is the case there should be no problem.
The question is, are there RelVal workflows where this is not the case?
I'm happy to work on a patch asap, but I would need inputs to understand better how I should be testing it, as my check with `runTheMatrix.py -l limited -i all` clearly failed to spot this issue, as well as the one with `TauAnalysis/MCEmbeddingTools/` that was instead spot during integration ...

Reply:
@battibass indeed this is a bit special case, and only wf 1102.0 implementing it shows this issue. I see that @slava77 has already mentioned it in https://github.com/cms-sw/cmssw/issues/26500 .